projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -1502,17 +1571,33 @@ export class IgxColumnGroupComponent extends IgxColumnComponent implements After
      * @memberof IgxColumnGroupComponent
      */
     get width() {
-        let isChildrenWidthInPercent = false;
-        const width = `${this.children.reduce((acc, val) => {
-            if (val.hidden) {
-                return acc;
-            }
-
-            if (typeof val.width === 'string' && val.width.indexOf('%') !== -1) {
-                isChildrenWidthInPercent = true;
-            }
-            return acc + parseInt(val.width, 10);
-        }, 0)}`;
+        let isChildrenWidthInPercent = false, width;
+        if (this.grid && this.grid.enableMRL) {
```

Comment:
this getter is probably used a lot and the calculation doesn't look very quick - consider caching

Reply:
I also see the two if branches are quite similar so you can reuse more code
this might not be an issue if we extend the column group component for MRL